packages/fields/src/types/Text/views/Controller.js

```diff
@@ -2,9 +2,10 @@ import FieldController from '../../../Controller';
 
 export default class TextController extends FieldController {
   getFilterGraphQL = ({ type, value }) => {
-    const key = type === 'is' ? `${this.path}` : `${this.path}_${type}`;
```

Comment:
I would be happy with this file only changes in the PR (+ test fix if any)

Reply:
@gautamsi What do you mean by " with this file only changes".  You mean this pull request should only make changes to this file?

Reply:
to support insensitive search with minimal changes, changes proposed in this file is sufficient.

To avoid long delay you can create a normal PR (not WIP) with this this file change and fix the test cases, hopefully that goes through fast

Reply:
fresh PR is probably better with new changeset